WordPress/src/main/res/values/strings.xml

```diff
@@ -1037,6 +1037,7 @@
     <string name="activity_log_item_menu_download_backup_label">Download backup</string>
 
     <!-- scan -->
+    <string name="backup">Backup</string>
```

Comment:
Minor 🔍: We might want to put `Backup` related strings in a separate section.

Reply:
This is now done: 87590d6338a2c676f8860eafe5e818a595e6f9af

Thanks for point it out Ashita! ❤️